site2/docs/functions-configure.md

```diff
@@ -0,0 +1,189 @@
+# Pulsar Functions configure
```

Comment:
I am not sure if this is a right heading. This page seems to be related to functions command-line tools?

Reply:
Yes, and we can rename it as "Pulsar Functions CLI", and add another topic "Pulsar Functions config" on how to configure the YAML file.